docs/docusaurus/docs/how_to_get_support.md

```diff
@@ -0,0 +1,3 @@
+---
+title: How to get support
```

Comment:
This file will contain the unified info for support that will result of this documentation JIRA ticket: https://greatexpectations.atlassian.net/browse/DOC-649

I leave it here as a placeholder, but if by the time this PR is reviewed we don't have it yet, I can just hide the link in the navbar and add it later!

Reply:
File is already merged so I'm deleting this one and redirecting to the existing one!